css/navigation.css

```diff
@@ -1 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
 
+nav {
+  display: flex;
+  justify-content: center;
+  font-size: 17px;
+  border-style: solid;
+  border: #eeeeee;
+}
+
+ul {
+  list-style-type: none;
+  font-family: "Courier New", Courier, monospace;
+}
+
+a {
+  text-decoration: none;
+  padding: 10px;
+}
```

Comment:
If you don't use `text-decoration` for your links, you should add an hover effect with `a:hover` to indicate that they are clickable.